### PR TITLE
Update Notice Module and Module CSS loading API

### PR DESCRIPTION
**Pull request by kabel**
* Reverts bad commit to debug html instance
* Changes the example markup for notice
* Updates the core band styles to use margin to offset the page-pad on mobile
* Updates notice styles to not do so many overrides
* Cleanup WDN api to remove dead code and add a method for getting a versioned template file URL
* Update all modules to use the versioned template file URL getter for loading CSS.

### CODE REVIEW

File: wdn/templates_4.0/less/layouts/bands.less [hunk 1]
```diff
@@ -9,17 +15,25 @@
...
     .wdn-inner-wrapper {
         .clear-fix();
-        padding: 0 @mobile-pad;
+        padding-left: @mobile-pad;
+        padding-right: @mobile-pad;
 
         @media @bp3 {
-            padding: 0;
+            padding-left: 0;
+            paading-right: 0;
```
Comment (mfairchild365) [inline wdn/templates_4.0/less/layouts/bands.less]: Typo?

Comment (kabel) [inline wdn/templates_4.0/less/layouts/bands.less]: Indeed.

File: wdn/templates_4.0/examples/notice.html [hunk 2]
```diff
@@ -41,28 +38,31 @@
...
+        <p class="title">Success!</p>
+        <p>This notice overlays the maincontent and will vanish on its own in 15 seconds.</p>
     </div>
 </div>
+<script type="text/javascript">
+WDN.initializePlugin('notice');
+</script>
 </div>
 </body>
 </html>
```
Comment (mfairchild365) [inline wdn/templates_4.0/examples/notice.html]: I wonder if these examples could use some accessibility love.  Perhaps aria roles of `alert` and/or `alertdialog`.  I also wonder if it is possible to have the JavaScript side of things add the appropriate role automatically, so it doesn't have to be hand coded.

Comment (kabel) [inline wdn/templates_4.0/examples/notice.html]: They _could_, but this is beyond the scope of this PR.

Comment (mfairchild365) [inline wdn/templates_4.0/examples/notice.html]: Is it beyond the scope?  You are already refactoring it, accessibility should be in mind.

I am curious exactly how AT processes these roles if they exist on the page on initial page load.  Do they announce them right away?  I will have to check.

Comment (kabel) [inline wdn/templates_4.0/examples/notice.html]: That's like saying, "hey, I see you changing something in this file, mind changing something else mostly unrelated to your original intent?"

Make you own PR. :P

Comment (mfairchild365) [inline wdn/templates_4.0/examples/notice.html]: Sorry, I don't see how accessibility is mostly unrelated.  The example code was updated to include the latest best practices, which should include accessibility considerations.

Comment (kabel) [inline wdn/templates_4.0/examples/notice.html]: I'm not trying to be confrontational, but this is exactly what feature creep is. If you see something else that needs improvement, you should create your own pull request.

Comment (mfairchild365) [inline wdn/templates_4.0/examples/notice.html]: I'm not trying to be confrontational either.  I'd argue that feature creep is not this.  Pull requests should be reviewed for code style, bugs, usability and risk.  Risk and usability includes accessibility.  If care was taken to refactor something to make it better (and this is much better than it was before), accessibility should be reviewed in the pull request.

Feature creep on the other hand is adding additional superficial (maybe the wrong word) features, such as different fading transitions, or different colors because they would look nice, or a different widget entirely.

When a widget is updated, all aspects of that widget in terms of code and usability should be visited and addressed.

Perhaps that is just my professional opinion.

**Comment by mfairchild365** [comment]
@kabel it looks like the tests are failing for this one.

**Comment by kabel** [comment]
Boo. It seems like a side-effect. I'm restarting the build to confirm.

**Comment by mfairchild365** [comment]
Looks like it is still failing. :(

**Comment by kabel** [comment]
Found it.
